stable/netsil/README.md

```diff
@@ -0,0 +1,44 @@
+# Netsil Chart
+
+[Netsil](https://netsil.com/) provides universal observability.
+
+## Chart Details
+
+* Installs a deployment that provisions the Netsil AOC Console along with a DaemonSet for the AOC agent.
+* Requires 2X Large sizes at AWS or equivalent.
+* Requires the [kube-state-metrics chart](https://github.com/kubernetes/charts/tree/master/stable/kube-state-metrics).
+
+## Prerequisites
+
+- Kubernetes 1.5+
+- 2X Large Instances used for hosts running the console
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```bash
+$ helm install --name my-release stable/netsil
+```
+
+## Configuration
+
+The following tables lists the configurable parameters of the Netsil chart and their default values.
+
+|      Parameter            |          Description            |                         Default                         |
+|---------------------------|---------------------------------|---------------------------------------------------------|
```

Comment:
Please update the table to correctly reflect the `values.yaml` and align the table properly.